### DISCUSSION TITLE
[M-Z] (2024/25) - sezione 0.7 es 4.21 (30 ottobre)

### DISCUSSION THREAD

**Discussion by erinrrr** [sapienzastudentsnetwork/calcolo-delle-probabilita#311] [Esercizi M-Z] [answered]
![image](https://github.com/user-attachments/assets/b14ca0ae-ba2a-41f5-aaf6-84224b220c42)

**Comment by erinrrr** [comment, marked as answer]
![image](https://github.com/user-attachments/assets/c7b80aed-a74b-4632-8966-78517187c3d8)

**Comment by marcoseverini** [comment]
![immagine](https://github.com/user-attachments/assets/04b28506-e39e-4f98-bb2b-c79f367077c3)
